tests/test_RecursiveFetaureElimination.py

```diff
@@ -1,6 +1,98 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.datasets import make_friedman1
+from sklearn.linear_model import LinearRegression
+
 from feature_selection import recursive_feature_elimination
 
 
-def test_1():
-    result = recursive_feature_elimination(1, 2, 3, 4, 5)
-    assert result == []
+def scorer(X, y):
+    """
+    Sample custom scorer that fits a model and returns
+    an appropriate score for the feature selection problem.
+
+    Parameters
+    ----------
+    X : array-like of shape (n_samples, n_features)
+        Test samples
+
+    y : array-like of shape (n_samples,) or (n_samples, n_outputs)
+        True values for X, used for training
+
+    Returns
+    -------
+    Index of feature column with the lowest weight.
+    """
+    model = LinearRegression()
+    model.fit(X, y)
+    return X.columns[model.coef_.argmin()]
```

Comment:
See prior comments regarding scorer in RecursiveFeatureElimination.py

Reply:
Keep in mind that the scorer function needs to be implemented by the end-user. This is just a sample scorer. It's up to the end-user to create a scorer function that is appropriate to the model being used.